documentation/content/en/articles/committers-guide/_index.adoc

```diff
@@ -1555,9 +1556,9 @@ Here's https://adventurist.me/posts/00296[a good writeup] that goes into more de
 
 ==== Developers
 
-===== Ooops! I committed to `main` instead of a branch.
+===== Ooops! I committed to `main`.
```

Comment:
We can make it better, and keep it fairly colloquial, however: 

* the current wording wrongly implies that `main` is not a branch.

Reply:
Fixed with <[`715da80` (#182)](https://github.com/freebsd/freebsd-doc/pull/182/commits/715da80916b297d253451e278e20ba6b22f2ca29)>: 

> ===== Ooops! I committed to `main`, instead of another branch.